cmd/fs-v1.go

```diff
@@ -853,10 +854,6 @@ func (fs *FSObjects) putObject(ctx context.Context, bucket string, object string
 		return fsMeta.ToObjectInfo(bucket, object, fi), nil
 	}
 
-	if err = checkPutObjectArgs(ctx, bucket, object, fs, data.Size()); err != nil {
```

Comment:
object args need to be checked  for destination object in fs CopyObject since you have commented this out.

Reply:
Added :+1:  PTAL